include/LightGBM/tree.h

```diff
@@ -151,7 +151,7 @@ class Tree {
   /*! \brief Get the number of data points that fall at or below this node*/
   inline int data_count(int node) const { return node >= 0 ? internal_count_[node] : leaf_count_[~node]; }
 
-  
+
```

Comment:
I think this line can be removed as we usually use 1 blank line between methods.

Reply:
even better!